app/presenters/teams/filtered_list.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module Teams
+  class FilteredList < ::FilteredList
+
+    def fetch_entries
+      if query.present?
+        return teams.where('name like ?', "%#{query}%")
+      end
+
+      teams
+    end
+
+    def query
+      @params[:q]
+    end
+
+    def team_ids
+      @params[:team_ids]
```

Comment:
use list_param .... 
add new method list_param to base filtered class and use it in here

``
def list_param(key)
  params[key].to_s.split(',').map(&:strip)
end
``